configurations/jupyter/migrated

```diff
@@ -0,0 +1 @@
+2019-08-26T14:02:30.274047
```

Comment:
I do not think this file is needed?!

Reply:
Added it to .gitignore. It is automatically created.